### PR TITLE
Add vaccination didcomm message examples

### PR DESCRIPTION
**Pull request by OR13**
This PR addresses:

- #18
- #20 
- #21
- #22

@csuwildcat @brentzundel Would love your help filling in any gaps int he presentation defintion object... but it might be easier to tackle in a subsequent PR.

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -511,6 +511,180 @@ sequenceDiagram
...
+  "to": "did:example:prover",
+  "body": {},
+  "attachments": [
+    {
+      "@id": "2a3f1c4c-623c-44e6-b159-179048c51260",
+      "mime-type": "application/ld+json",
+      "format": "dif/presentation-exchange/submission@v1.0",
+      "data": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1"
```
Comment (JaceHensley) [inline spec/spec.md:624]: Does this need the Presentation Submission context and type entries?

**Review by JaceHensley — CHANGES_REQUESTED** [review]
Just one question, looks good otherwise

File: spec/spec.md (hunk 1 above)
Comment (OR13) [reply spec/spec.md:624]: excellent catch, you are correct.

Comment (OR13) [inline spec/spec.md:624]:
```suggestion
          "https://www.w3.org/2018/credentials/v1",
          "https://identity.foundation/presentation-exchange/submission/v1"
```

**Review by brentzundel — APPROVED** [review]

File: spec/spec.md [hunk 2]
```diff
@@ -511,6 +511,180 @@ sequenceDiagram
...
+        ],
+        "type": [
+          "VerifiablePresentation"
```
Comment (OR13) [inline spec/spec.md:628]:
```suggestion
          "VerifiablePresentation",
          "PresentationSubmission"
```

**Review by JaceHensley — APPROVED** [review]

**Pull request merged**